dev/NavigationView/TestUI/NavigationViewTopNavOnlyPage.xaml

```diff
@@ -29,7 +29,7 @@
                 <muxcontrols:NavigationViewItem x:Name="HomeItem" Content="Home" Icon="Home" />
                 <muxcontrols:NavigationViewItem x:Name="AppsItem" Icon="Shop" Tag="Apps" /> <!-- AppsItem only have Icon and no content -->
                 <muxcontrols:NavigationViewItem x:Name="SuppressSelection" Content="SuppressSelection" Icon="Emoji" SelectsOnInvoked="False" />
-                <muxcontrols:NavigationViewItem x:Name="GamesItem" Content="Games" Icon="Emoji" />
+                <muxcontrols:NavigationViewItem x:Name="GamesItem" Content="Should be green" Icon="Emoji" Foreground="Green"/>
```

Comment:
Hmm, so at first glance, I see why this change would be failing the`TopNavigationOverflowWidthLongNavItemTest` (it counts items being moved and changing the content changed the length of the item). I'm curious as to why RS5 and above is not failing. Looking into it.

As a quick fix, I think moving your test of this change to another test page (such as 'NavigationViewTopNavPage.xaml') should avoid this issue (just make sure that item you are updating is not being looked for in other tests).

Reply:
Great idea, moved it to a different page, where this should not make a difference. Thanks for you help so far :)

Reply:
awesome, looks like it worked. Thanks for making the update!

Reply:
@ojhad  Any ideas why this would cause a difference in behavior on different platforms ?